docker_custodian/docker_gc.py

```diff
@@ -127,21 +130,23 @@ def remove_image(client, image_summary, min_date, dry_run):
     image_tags = image_summary.get('RepoTags')
     # If there are no tags, remove the id
     if no_image_tags(image_tags):
-        api_call(client.remove_image, image_summary['Id'])
+        api_call(client.remove_image, image=image_summary['Id'])
         return
 
     # Remove any repository tags so we don't hit 409 Conflict
     for image_tag in image_tags:
-        api_call(client.remove_image, image_tag)
+        api_call(client.remove_image, image=image_tag)
 
 
-def api_call(func, id):
+def api_call(func, **kwargs):
     try:
-        return func(id)
+        return func(**kwargs)
     except requests.exceptions.Timeout as e:
-        log.warn("Failed to call %s %s %s" % (func.__name__, id, e))
+        params = ','.join(['%s=%s' % (k, v) for k, v in kwargs.items()])
```

Comment:
minor nit: you don't actually have to split this into `k, v` here, and the list comprehension is unnecessary, if you remove the `[]` it will be a generator comprehension

```python
params = ','.join('%s=%s' % item for item in kwargs.items())
```